controls/include/controls/six_deg_PID.h

```diff
@@ -0,0 +1,44 @@
+//
+// Created by tanvi on 11/21/2020.
+//
+
+#ifndef CONTROLS_SIX_DEG_PID_H
+#define CONTROLS_SIX_DEG_PID_H
+#include "PID.h"
+#include <vector>
+
+class six_deg_PID
+{
+private:
+    PID roll_ = PID();
+    PID pitch_ = PID();
+    PID yaw_ = PID();
+
+    PID x_ = PID();
+    PID y_ = PID();
+    PID z_ = PID();
+
+public:
+
+    six_deg_PID(std::vector<double> kP, std::vector<double> kI, std::vector<double> kD);
+
+    PID getRoll();
+    PID getPitch();
+    PID getYaw();
+
+    PID getX();
+    PID getY();
+    PID getZ();
+
+    void tuneAngle(std::vector<double> kP, std::vector<double> kI, std::vector<double> kD);
+
+    void tunePosition(std::vector<double> kP, std::vector<double> kI, std::vector<double> kD);

```

Comment:
In my opinion this interface may be tricky to work with. In general, we tune one PID loop at a time. So, it may be a bit inconvenient to make these vectors of coefficients. I would recommend either having six functions, one per controller, or just one which takes the loop to tune as an argument (probably passed by reference). So the options would be:

```
void tuneSurge(double kP, double kI, double kD);
void tuneSway(double kP, double kI, double kD);
...
```

OR 

`void tuneController(double kP, double kI, double kD, const PID& controller)`